Framework/LiveData/test/LoadLiveDataTest.h

```diff
@@ -180,8 +180,8 @@ class LoadLiveDataTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(ws1->getNumberHistograms(), 2);
     double total;
     total = 0;
-    for (auto it = ws1->readY(0).begin(); it != ws1->readY(0).end(); it++)
-      total += *it;
+    for (double it : ws1->readY(0))
```

Comment:
Could `it` be renamed something like `yValue`? Also could 184 be put inside braces? The same goes for line 194